src/wallet/crypter.cpp

```diff
@@ -88,19 +91,20 @@ bool CCrypter::Encrypt(const CKeyingMaterial& vchPlaintext, std::vector<unsigned
     return true;
 }
 
-bool CCrypter::Decrypt(const std::vector<unsigned char>& vchCiphertext, CKeyingMaterial& vchPlaintext) const
+bool CCrypter::Decrypt(const std::vector<unsigned char>& ciphertext, CKeyingMaterial& plaintext) const
 {
     if (!fKeySet)
         return false;
 
     // plaintext will always be equal to or lesser than length of ciphertext
-    vchPlaintext.resize(vchCiphertext.size());
+    plaintext.resize(ciphertext.size());
 
     AES256CBCDecrypt dec(vchKey.data(), vchIV.data(), true);
-    int nLen = dec.Decrypt(vchCiphertext.data(), vchCiphertext.size(), vchPlaintext.data());
-    if(nLen == 0)
+    int len = dec.Decrypt(ciphertext.data(), ciphertext.size(), plaintext.data());
```

Comment:
nit to reduce variable scope and make it slightly more readable:

<details>
<summary>git diff on bad1ffb725</summary>

```diff
diff --git a/src/wallet/crypter.cpp b/src/wallet/crypter.cpp
index a574c8909e..a4b1fd9b3b 100644
--- a/src/wallet/crypter.cpp
+++ b/src/wallet/crypter.cpp
@@ -100,12 +100,11 @@ bool CCrypter::Decrypt(const std::vector<unsigned char>& ciphertext, CKeyingMate
     plaintext.resize(ciphertext.size());
 
     AES256CBCDecrypt dec(vchKey.data(), vchIV.data(), true);
-    int len = dec.Decrypt(ciphertext.data(), ciphertext.size(), plaintext.data());
-    if (len == 0) {
-        return false;
+    if (int len{dec.Decrypt(ciphertext.data(), ciphertext.size(), plaintext.data())}) {
+        plaintext.resize(len);
+        return true;
     }
-    plaintext.resize(len);
-    return true;
+    return false;
 }
 
 bool EncryptSecret(const CKeyingMaterial& vMasterKey, const CKeyingMaterial &vchPlaintext, const uint256& nIV, std::vector<unsigned char> &vchCiphertext)

```
</details>

Reply:
Started changing it based on your comment, but as there is already another if-return false above in the untouched code, I think it's more consistent to keep as-is.